PIL/TiffImagePlugin.py

```diff
@@ -549,7 +549,9 @@ def save(self, fp):
                 # and doesn't match the tiff spec: 8-bit byte that
                 # contains a 7-bit ASCII code; the last byte must be
                 # NUL (binary zero). Also, I don't think this was well
-                # exercised before.
+                # excersized before.
```

Comment:
Looks like the conflict was resolved the wrong way, reversing the effect of PR #1311